galaxy/importer/loaders/role.py

```diff
@@ -52,8 +52,9 @@ class RoleMetaParser(object):
         'youtube': 'https://www.youtube.com/embed/{0}',
     }
 
-    def __init__(self, metadata, logger=None):
+    def __init__(self, metadata, dependencies, logger=None):
         self.metadata = metadata
+        self.dependencies = dependencies
```

Comment:
If self.dependencies needs to always be list-like, I would use a optional arg and set it to empty list if None

something like

``` python
def __init__(self, metadata, dependencies=None, logger=None):
    self.metadata = metadata
    # or possible a if self.dependencies is None if it could potentially be Truthy (True, 1, 'nonempty-string", etc)
    self.dependencies = dependencies or []
 ```

Reply:
Good idea. Fixed.